vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -2292,6 +2317,26 @@ def zmq_ctx(socket_type: Any, addr: str) -> Iterator[zmq.Socket]:
             ctx.destroy(linger=0)
 
 
+def yield_req_data(
+    scheduler_output,
+) -> Iterator[tuple[str, tuple[list[int], ...], bool]]:
```

Comment:
I don't see harm in moving this shared function in utils

Reply:
done